akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -336,6 +339,7 @@ private final class Sdk(
 
   private val logger = LoggerFactory.getLogger(getClass)
   private val serializer = new JsonSerializer
+  private val asyncUtils = new RetriesImpl(system.classicSystem)
```

Comment:
`retries`. not that it is heavy but could probably be lazy because will most often not be used

Reply:
fixed